### ISSUE TITLE
Interface throws error about non valid float value

### ISSUE THREAD

**Issue by mazes-80** [fluxionary/minetest-choppy#7]
`2023-03-07 14:09:42: ERROR[Main]: Invalid float setting in .minetest/mods/choppy/settingtypes.txt "choppy.dig_speed_multiplier (dig speed multiplier) float 1 0.0000000001 inf"`

I think it just need to change inf to very high value

**Comment by fluxionary** [comment]
ha, i don't think i realized that the engine even had a validator for that file. easy fix.

**Comment by fluxionary** [comment]
@mazes-80 should be good w/ the most recent commit, do you want to verify it?

**Comment by fluxionary** [comment]
upstream issue: https://github.com/minetest/minetest/issues/13301

**Comment by mazes-80** [comment]
I was surprised to discover this error as I just downloaded this mod, not even tried it yet. This is part of my process for a new feature in my version of maidroid. They may end up having a lumberjack job, with this mod or lumberjack mod. I excluded chainsaw being part of possible tools now.

About checking, I will not bother to as I see no reason your commit does not fix this.
Glad to see it ended in a bug on minetest tracker.
